src/Core/Util/DateTime/NullDateTime.php

```diff
@@ -201,7 +201,7 @@ public function __wakeup()
     /**
      * {@inheritdoc}
      */
-    public static function createFromInterface(DateTimeInterface $object)
+    public static function createFromInterface(DateTimeInterface $object): DateTimeImmutable
```

Comment:
Are you sure the strongly typed method is compatible with previous PHP versions? The typing was not explicit before I'm afraid of a conflict

Reply:
>Does it mean the simplexml_load_file returns is different in PHP 8.0? Or does it mean it was never right? 😅

It was never right 😅

>Are you sure the strongly typed method is compatible with previous PHP versions? The typing was not explicit before I'm afraid of a conflict

Yeah, it is compatible!